pkg/ui/src/views/shared/components/loading/index.tsx

```diff
@@ -12,11 +12,12 @@ interface LoadingProps {
 // * loading prop is true.
 // *
 export default function Loading(props: LoadingProps) {
+  const className = props.className || "loading-image loading-image__spinner-left";
```

Comment:
Nice.  Heck, why not `const imagePath = props.image || spinner;`, too?

Reply:
Done. Hilarious how many call sites pass the same thing, and that we didn't do this earlier 😂